main.py

```diff
@@ -0,0 +1,8 @@
+from neuraxle.distributed import RestWorker
+
+
+if __name__ == '__main__':
+    worker = RestWorker(port=5000)
+    worker.start(upload_folder='/home/alexandre/Documents/cluster')
+    while True:
+        pass
```

Comment:
Is this loop needed? I thought that when a WSGI / Flask server was started, that it captured the main thread. I would make the worker capture the main thread as such, thus it would remove the need of doing a while true busy wait loop here.

Reply:
dead code for testing purposes